subjects/devops/kept-promise/README.md

```diff
@@ -0,0 +1,56 @@
+## kept-promise
+
+### Instructions
+
+Things in your code do not always happen as fast as you would like.
+
+Sometimes it is important to wait for a function to finish before keep going on. Other times you don't want your code to be blocking while waiting.
+
+Create an asynchronous function `processInfo` in a file `kept-promise.js`. The function `processInfo` will take an asynchronous function as an input, and it will print the message `Ok!` if the return of the input function is zero or a multiple of two, `Error!` otherwise.
```

Comment:
It would be nice to have a hint explaining what is an async function, and provide an example, in a similar way as in most exercises in the scripting piscine. For instance:
https://github.com/01-edu/public/tree/master/subjects/devops/burial

Reply:
It would also be nice to explain what does asynchronous mean in this context.

Reply:
I feel there are many examples of asynchronous processes now with the `Promise`, `await` and `then` examples. As a general concept, I feel the link to the general documentation is the best to explain this complex concept - if you have a nice, brief way to explain this to complement the official documentation, please, write it here :)

Reply:
What would you think about this:
" Asynchronous in the context of server-side JavaScript refers to the ability of code to run non-blockingly, meaning for instance that the server can handle multiple requests simultaneously without waiting for a long-running task to complete, avoiding to block the  server responsiveness. It is essential to achieve scalable and functional back-end applications in Node.js."